student.rb

```diff
@@ -2,6 +2,8 @@
 
 # This Student class can be initialised by passing classroom, Age, Name & parent permission.
 class Student < Person
+  attr_accessor :classroom
+
```

Comment:
- [ ] only getter is required for `classroom` 
- [ ] It should a method that serves as the 'belongs-to' side (a student belongs to a classroom)
```ruby

  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```